bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: Add new strimzi section for data plane cluster status

## Description
Update reported strimzi versions format to use new format that will be reported by KAS fleetshard operator in a future release.

Related to: https://issues.redhat.com/browse/MGDSTRM-4384

For now both formats have to be maintained and accepted until a new KAS Fleet Shard operator version that removes the old format is released and deployed to stage and production in the future.

## Verification Steps

run unit and integration tests.

A manual verification is running kas fleet manager creating a cluster and checking that the stored format looks like this:
```
[{"ready": true, "version": "strimzi-cluster-operator.v0.22.1-6"}, {"ready": true, "version": "strimzi-cluster-operator.v0.23.0-0"}]
```

Also another possible desired manual is to store the versions in the old format manually with SQL where  `available_strimzi_versions` of a cluster entry and then verify that it is automatically changed to the new format without errors.

## Checklist (Definition of Done)
